imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -63,47 +79,57 @@ class CaseExplorer extends Component {
   }
   render () {
     const { isLoading, dispatch, match } = this.props
-    const { unitsDict } = this.state
-
+    const { unitsDict, status, expandedUnits } = this.state
     return (
       <div className='flex flex-column roboto overflow-hidden flex-grow h-100'>
         <div className='bb b--black-10 overflow-auto flex-grow'>
+          <div className='flex pl3 pv3 bb b--very-light-gray'>
+            <div onClick={() => this.handleStatusClicked(0)} className={'f6 fw5 ' + (status === 0 ? 'mid-gray' : 'silver')}> Open </div>
```

Comment:
Since you defined an object that contains an "enum" like mapping (status), you should have used that for the values sent by `handleStatusClicked`. It makes the code easier to read